functions/typescript-with-node/handler.ts

```diff
@@ -0,0 +1,9 @@
+export {handle};
+
+function handle (event, context, cb) {
```

Comment:
This isn't really TypeScript haha. But I think for now it's fine, we should revisit this once we have finished `scaleway-functions-node` for Node and added type information to event/context.

I would still add a small type info to make this code non-valid Javascript (otherwise you don't need to use `tsc`):

```suggestion
function handle (event: Record<string, unknown>, context: Record<string, unknown>, cb: unknown) {
```

(Of course this type info is a bit useless haha so maybe we can find something better :smiling_face_with_tear: )